pywbemtools/pywbemcli/pywbemcli.py

```diff
@@ -369,7 +369,7 @@ def set_default_if_empty_str(tst_str, default=None):
               default=None,
               help=u'Display extra information about the processing.')
 @click.option('--warn/--no-warn', is_flag=True,
-              default=False,
+              default=None,
```

Comment:
Complex question.  If it is included in context object, there should be a None to indicate that there was no input since --warn and --no-warn represent option input and option state for this command The only reason we include it in the context object is because in interactive mode if you change it that should only apply to that single command so need the original in the top level context object (Note that need only be in the top level context object in reality but since click builds multiple levels, it was easy to include it in each context object.  Therefore, we need to know  what the original setting was and fall back to that after each interactive mode command.